docker-gc

```diff
@@ -190,11 +191,18 @@ xargs -n 1 $DOCKER inspect -f '{{.Id}}' 2>/dev/null |
 sort | uniq > images.used
 
 # List images to reap; images that existed last run and are not in use.
-$DOCKER images -q --no-trunc | sort | uniq > images.all
+echo -n "" > images.all
+$DOCKER images | while read line
+do
+    awk '{print $1};'
```

Comment:
I think this while loop with awk could be replaced with `$DOCKER images | cut -d " " -f 1`.

Also you may want to filter out the value of `<none>` for images that have no tags.